src/docs/product/issues/issue-owners/index.mdx

```diff
@@ -58,6 +58,41 @@ You can also add a rule from the Issue Details page:
 
 When you create a rule from the Issue Details page, you'll see some suggested paths and URLs based on the events in the issue. Note that [sentry.io](https://sentry.io) doesn't suggest tags.
 
+### Code Owners
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+Let us know if you have feedback: [ecosystem-feedback@sentry.io](mailto:ecosystem-feedback@sentry.io).
+
+</Note>
+
+![Code Owners in UI](code_owners.png)
+
+Import your CODEOWNERS file and use them alongside your Ownership Rules to assign Sentry Issues. You can import and incrementally add the mappings between your source control teams/users to Sentry Teams/Users using [External Team/User Mappings](/product/issues/issue-owners/#external-teamuser-mappings). We will automatically ignore rules that are missing team/user mappings.
```

Comment:
"them" is unclear here - does it refer to the file (in which case, the edit would be to "use it") or the rules? I suspect it's the file, so editing to:
```suggestion
Import your CODEOWNERS file and use it alongside your Ownership Rules to assign Sentry issues. You can import and incrementally add the mappings between your source control teams/users to Sentry Teams/Users using [External Team/User Mappings](/product/issues/issue-owners/#external-teamuser-mappings). Sentry will automatically ignore rules that are missing team/user mappings.
```